code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -4,7 +4,7 @@
 	var/obj/machinery/camera/current = null
 	icon = 'icons/mob/pai.dmi'
 	icon_state = "repairbot"
-	mouse_opacity = 1
+	mouse_opacity = 2

```

Comment:
This makes clicking anything that's not pAI impossible. I guess it's not a big difference but it feels lazy.